agent/stats/common_test.go

```diff
@@ -151,6 +152,9 @@ func validateInstanceMetrics(t *testing.T, engine *DockerStatsEngine) {
 	assert.Equal(t, aws.StringValue(taskMetric.TaskDefinitionFamily), taskDefinitionFamily, "unexpected task definition family")
 	assert.Equal(t, aws.StringValue(taskMetric.TaskDefinitionVersion), taskDefinitionVersion, "unexpected task definition version")
 	assert.NoError(t, validateContainerMetrics(taskMetric.ContainerMetrics, 1), "validating container metrics failed")
+	if includeServiceConnectStats {
+		assert.NoError(t, validateServiceConnectMetrics(taskMetric.ServiceConnectMetricsWrapper, 1), "validating service connect metrics failed")
```

Comment:
Should we include the exact error in messages?
"validating service connect metrics failed: %v", err

Reply:
if there is an error, the error message shows up in the logs so we do not have to include the error message explicitly here.
```
        	Error Trace:	common_test.go:156
        	            				engine_unix_integ_test.go:128
        	Error:      	Received unexpected error:
        	            	Mismatch in number of serviceConnectMetrics elements. Expected: 1, Got: 0
        	Test:       	TestStatsEngineWithServiceConnectMetrics
```